sdk/servicebus/service-bus/src/core/messageSender.ts

```diff
@@ -589,7 +593,7 @@ export class MessageSender extends LinkEntity {
    * Batch message.
    * @return {Promise<void>}
    */
-  async sendBatch(inputMessages: ServiceBusMessage[]): Promise<void> {
+  async sendArrayOfMessages(inputMessages: ServiceBusMessage[]): Promise<void> {
```

Comment:
Why not just `sendMessages`?

Reply:
I had `sendMessages` in mind before going for `sendArrayOfMessages`.
Just to avoid confusion later on, I picked `ArrayOfMessages`.

I can rollback if you strongly feel for it.

Reply:
For consistency it seems reasonable to just name it sendMessages.